content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
+Wooh!
+
+Manubot applies **tactics from open source** software to manuscript writing.
```

Comment:
Suggest: `Manubot is based on a collection of tools from the open source ecosystem`

Reply:
This is true, but was not the point I was trying to get across. The point is that we adopt the workflow of OSS:

+ writing is done publicly under an open license
+ additions are subject to review
+ contributions are traceable
+ everything is scripted and automated / continuous integration
+ identifiers

Reply:
Ahh! I missed that. Perhaps this would be clearer:
> Manubot transfers the lessons from the open source software movement to academic writing.

And then you could name some of those features here in addition to talking about the technologies you are using.

Reply:
Okay how about https://github.com/greenelab/manufund-2018/pull/2/commits/dd761d614e0a9c9f67c909cef9e1ce0a89f44d26?